libnmstate/state.py

```diff
@@ -88,6 +115,20 @@ def _complement_interface_empty_ip_subtrees(self):
                 if family not in iface_state:
                     iface_state[family] = {}
 
+    @property
+    def routes(self):
```

Comment:
How will you call the 'running' entries if you reserve the base name to config?

Reply:
We can rename it when we need running state which I doubt it.